usaspending_api/etl/management/commands/load_multiple_submissions.py

```diff
@@ -1,126 +1,138 @@
 import logging
-import pytz
 
-from datetime import datetime
+from datetime import timedelta
+from django.conf import settings
 from django.core.management import call_command
 from django.core.management.base import BaseCommand
-from django.db import connections, DEFAULT_DB_ALIAS
+from django.db import connection
+from django.db.models import Max
+from usaspending_api.submissions.models import SubmissionAttributes
 
-logger = logging.getLogger("console")
-exception_logger = logging.getLogger("exceptions")
 
+logger = logging.getLogger("script")
 
-class Command(BaseCommand):
 
-    # Give it a fiscal year and a quarter. Will list missing subs/agencies and their recent certified dates.
+class Command(BaseCommand):
     def add_arguments(self, parser):
-        parser.add_argument("fy", nargs=1, help="the fiscal year", type=int)
-        parser.add_argument("quarter", nargs=1, help="the fiscal quarter to load", type=int)
-        parser.add_argument("--safe", action="store_true", help="only list missing submissions from the FY/Quarter")
-
-    def handle(self, *args, **options):
-
-        try:
-            broker_conn = connections["data_broker"]
-            broker_cursor = broker_conn.cursor()
-            api_conn = connections[DEFAULT_DB_ALIAS]
-            api_cursor = api_conn.cursor()
-        except Exception as err:
-            logger.critical("Could not connect to database(s).")
-            logger.critical(err)
-            return
-
-        fy = options["fy"][0]
-        quarter = options["quarter"][0]
-
-        if not 1 <= quarter <= 4:
-            logger.critical("Acceptable values for fiscal quarter are 1-4 (was {}).".format(quarter))
-            return
-
-        # Convert fiscal quarter to starting month of calendar quarter
-        quarter = int(quarter) * 3
-
-        logger.info("Querying Broker DB for Submissions")
-
-        broker_cursor.execute(
-            "SELECT submission.submission_id, MAX(certify_history.created_at) AS certified_at, \
-                                  submission.cgac_code, submission.frec_code \
-                                  FROM submission \
-                                  JOIN certify_history ON certify_history.submission_id = submission.submission_id \
-                                  WHERE submission.d2_submission = FALSE \
-                                  AND submission.publish_status_id IN (2, 3) \
-                                  AND submission.reporting_fiscal_year = {} \
-                                  AND submission.reporting_fiscal_period = {} \
-                                  GROUP BY submission.submission_id;".format(
-                fy, quarter
-            )
+        parser.add_argument(
+            "--submission-ids",
```

Comment:
Recent ETL scripts have an explicit incremental flag rather than default to this behavior. It helps everyone to run the scripts correctly.

Reply:
There will be another pull request today with the last of the DEV-5041 changes in a couple hours.  Ok if it implement it there instead of here?

Also, what would you like the default behavior to be if no switches are supplied?  Just bail?

Reply:
Quit the script. Could make the arguments mutually exclusive and required, that should handle the logic and error output